front/app/components/Form/Components/Layouts/CLPageLayout.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { memo, useState, useEffect, useContext } from 'react';
+import { LayoutProps, RankedTester, rankWith } from '@jsonforms/core';
+import { JsonFormsDispatch, withJsonFormsLayoutProps } from '@jsonforms/react';
+import styled, { useTheme } from 'styled-components';
+import Ajv from 'ajv';
+
+// Components
+import {
+  Box,
+  Title,
+  useBreakpoint,
+  media,
+} from '@citizenlab/cl2-component-library';
+import { FormSection } from 'components/UI/FormComponents';
+import Button from 'components/UI/Button';
```

Comment:
Are we be able to use the Button from the component library instead? Or is there a reason we need the components/UI one?

Reply:
Not really, this one also uses the library Button underneath. This particular instance is because of VSCODE auto import. Is there a reason we don't want to use this. Probably opens up another discussion on why the cl2 one doesn't support the things this one does